workflow-templates/upgrade-python-requirements.yml

```diff
@@ -2,67 +2,25 @@ name: Upgrade Requirements
 
 on:
   schedule:
-    # will start the job at midnight every monday (UTC), should be updated to use appropriate value
     - cron: "0 0 * * 1"
   workflow_dispatch:
     inputs:
       branch:
         description: "Target branch to create requirements PR against"
         required: true
-        default: 'master'
+        default: "master"
```

Comment:
you can use `$default_branch` here according to https://docs.github.com/en/actions/learn-github-actions/creating-workflow-templates